src/backend/distributed/deparser/objectaddress.c

```diff
@@ -0,0 +1,180 @@
+
```

Comment:
Missing file comment and function comments.

I can see that functions are pretty simple/straightforward and the names are descriptive. Still, it'd be good to stick to the conventions.